spec/child-apps/multiple-lifecycle-functions/multiple-lifecycle-functions.app.js

```diff
@@ -41,7 +41,7 @@ export const bootstrap = [
 			resolve();
 		});
 	},
-]
+];
```

Comment:
Yep webpack sucks really bad and needed these semicolons. Then I put in the babel-loader and it wasn't necessary to put in the semicolons anymore, but I figured the semicolons aren't hurting anything so I kept them here.